ci-infra/ami-build/scripts/aws-metadata-refresh.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
This is not a script I've evaluated. It was imported wholesale, and likely needs editing